### PR TITLE
Add MjGroup

### PR DESCRIPTION
**Pull request by epayet**
**Do not merge yet.**

This is related to [#41](https://github.com/mjmlio/mjml/issues/41). It adds the possibility to create groups and ensure that in the group, the columns will not stack on mobile.

Here is an example:

```html
<mj-body>
    <mj-section>
        <mj-group background-color="#cdd8da">
            <mj-column>
                <mj-text align="center">1</mj-text>
            </mj-column>
            <mj-column>
                <mj-text align="center">2</mj-text>
            </mj-column>
        </mj-group>

        <mj-group background-color="#9db3b6">
            <mj-column>
                <mj-text align="center">3</mj-text>
            </mj-column>
            <mj-column>
                <mj-text align="center">4</mj-text>
            </mj-column>
        </mj-group>

        <mj-group background-color="#6c8d91">
            <mj-column>
                <mj-text align="center">5</mj-text>
            </mj-column>
            <mj-column>
                <mj-text align="center">6</mj-text>
            </mj-column>
        </mj-group>
    </mj-section>
</mj-body>
```

Rendering in mobile:

mobile

Rendering on dekstop:

desktop

You will see that the Group component is very close to the Column component (there is even some copy/paste that can eventually be worth refactoring).

The trick is to put every children on the same `tr` sourrounded by `td` so that they stay on the same line.

It does not look good with other components like mj-button or mj-image, I am investigating and will update here. In the meantime, anybody can investigate as well ;)

Any thoughts?

### CODE REVIEW

**Comment by epayet** [comment]
Example that breaks:

```html
<mj-body>
    <mj-section>
        <mj-group background-color="#cdd8da">
            <mj-column>
                <mj-button align="center">1</mj-button>
            </mj-column>
            <mj-column>
                <mj-button align="center">2</mj-button>
            </mj-column>
        </mj-group>

        <mj-group background-color="#9db3b6">
            <mj-column>
                <mj-text align="center">3</mj-text>
            </mj-column>
            <mj-column>
                <mj-text align="center">4</mj-text>
            </mj-column>
        </mj-group>

        <mj-group background-color="#6c8d91">
            <mj-column>
                <mj-image src="http://media02.hongkiat.com/keep-calm-designer-posters/keep-calm-and-lorem-ipsum.jpg"/>
            </mj-column>
            <mj-column>
                <mj-image src="http://media02.hongkiat.com/keep-calm-designer-posters/keep-calm-and-lorem-ipsum.jpg"/>
            </mj-column>
        </mj-group>
    </mj-section>
</mj-body>
```

Rendering ok on mobile:

mobile

Rendering not ok on desktop:

desktop

WIP

File: src/components/Group.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    const mjColumnClass = this.getColumnClass()
+
+    this.styles = this.getStyles()
+
+    return (
+      <div style={this.styles.div} className={mjColumnClass} aria-labelledby={mjColumnClass} data-column-width={parseInt(width)}>
+        <table style={this.styles.table} data-legacy-background={mjAttribute('background')} width="100%">
+          <tbody>
+          <tr>
+            {this.renderChildren()}
```
Comment (iRyusa) [inline src/components/Group.js]: You might need to use `renderWrappedOutlookChildren` in order to keep outlook compatibility !

But it's looks really cool :+1:

**Comment by iRyusa** [comment]
Can you merge `1.3` branch on your branch ? big :+1: for it, i'll try to help on it

**Comment by epayet** [comment]
I just merged the 1.3-beta branch on this branch, sorry about the long git commit history now though since the target branch of this PR is the master. I welcome the help! Thanks

**Comment by iRyusa** [comment]
Yes, don't worry 1.3 should be merged on master today :+1:

**Comment by iRyusa** [comment]
Closing this one in favor of #281 ! Thanks for your work @epayet !

**Pull request closed without merging**